epsie/proposals/birthdeath.py

```diff
@@ -0,0 +1,315 @@
+# Copyright (C) 2020 Richard Stiskalek, Collin Capano
+
+# This program is free software; you can redistribute it and/or modify it # under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+from __future__ import (absolute_import, division)
+
+import itertools
+import math
+
+import random
+
+import numpy as np
```

Comment:
Thinking about this some more, I would prefer to keep it as `numpy`, to be consistent with the rest of the package.